visualization/motion_planning_rviz_plugin/src/motion_planning_frame_planning.cpp

```diff
@@ -129,15 +137,28 @@ void MotionPlanningFrame::computePlanButtonClicked()
 void MotionPlanningFrame::computeExecuteButtonClicked()
 {
   if (move_group_ && current_plan_)
-    move_group_->execute(*current_plan_);
```

Comment:
> Indeed. That looks promising. Could you prepare a PR for this?

This PR without [last commit](https://github.com/ros-planning/moveit_ros/pull/709/commits/5503fa925adbf5a795d4b5c7095117ccb1724f03) is that approach. The trajectory execution with execute_button on rviz plugin was able to be interrupted (with `asyncExecute()`).
However, it is unable to get the status of execution for service call, so I could not handle the GUI button status, enable/disable, correctly.